MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using MobileManager.Models.Devices;
+using MobileManager.Models.Devices.Interfaces;
+using MobileManager.SeleniumConfigs.DataSets.Interfaces;
+using DotLiquid;
+
+namespace MobileManager.SeleniumConfigs.DataSets
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// AndroidSeleniumConfig.
+    /// </summary>
+    public class AndroidSeleniumConfig : Drop, ISeleniumConfig

```

Comment:
what is "drop"?

Reply:
Drop is part of Liquid technology - https://github.com/dotliquid/dotliquid/wiki/DotLiquid-Drops